content/200-concepts/100-components/02-prisma-client/240-metrics.mdx

```diff
@@ -90,41 +92,41 @@ This returns metrics as follows:
       "description": "The total number of operations executed"
     },
     {
-      "key": "query_total_queries",
+      "key": "prisma_datasource_queries_total",
       "labels": {},
       "value": 0,
       "description": "The total number of queries executed"
```

Comment:
Did we not also change these descriptions @garrensmith?

Reply:
This has now been updated as part of the JSON dump @SevInf provided. Though the descriptions seem to be missing in this version.

Reply:
This was a Client bug, added in descriptions now.